File: Zadania/2025/Zadanie-5/Java/Klasa.java
package Java;

import java.util.ArrayList;
import java.util.List;

public class Klasa {
    String literaKlasy;
    int numerKlasy;
    List<Uczen>uczniowie;

    Klasa(String literaKlasy , int numerKlasy ,List<Uczen>uczniowie){
        this.literaKlasy = literaKlasy;
        this.numerKlasy = numerKlasy;
        this.uczniowie = new ArrayList<Uczen>();
    }
    
    public void dodajUcznia(Uczen uczen){
        uczniowie.add(uczen);
    }

}

File: Zadania/2025/Zadanie-5/Python/klasa.py
from uczen import Uczen

class Klasa:
    def __init__(self , literaKlasy , numerKlasy ):
        self.literaKlasy = literaKlasy
        self.numerKlasy = numerKlasy
        self.uczniowie = [Uczen]
    def dodajUcznia(self , uczen):
        self.uczniowie.append(uczen)